ui/selectors/selectors.js

```diff
@@ -662,7 +662,7 @@ export const getNonTestNetworks = createDeepEqualSelector(
           imageUrl: LINEA_MAINNET_TOKEN_IMAGE_URL,
         },
         providerType: NETWORK_TYPES.LINEA_MAINNET,
-        ticker: TEST_NETWORK_TICKER_MAP[NETWORK_TYPES.LINEA_MAINNET],
+        ticker: CURRENCY_SYMBOLS.ETH,
```

Comment:
the value of `TEST_NETWORK_TICKER_MAP[NETWORK_TYPES.LINEA_MAINNET]` is undefined

Reply:
Can we instead update `TEST_NETWORK_TICKER_MAP`? so it has something like:
`[NETWORK_TYPES.LINEA_MAINNET]: `Linea${CURRENCY_SYMBOLS.ETH}`,`